### PR TITLE
convert tutorial to jupyterbook

### PR DESCRIPTION
**Pull request by yibeichan**
Hello, this jupyterbook is ready for review https://yibeichan.github.io/pydra-tutorial/welcome.html

here is a list of what I did:
1. convert all our `.ipynb` to `myst-nb md` and pair them together, so we only need to modify `notebooks/*.md` and `.ipynb` will sync
2. create symlink in `jupyterbook` folder. Everytime we add or modify files in `notebooks`, `jupyterbook/notebooks` will change too. So does `figures`
3. `requirements.txt` updated
5.  create `About Pydra` section at the end of our jupyterbook, the content is TODO though.
6. put `hide-cell/hide-input/hide-output` settings when needed

Let me know if there are any details you want to change.

### CODE REVIEW

**Comment by effigies** [comment]
Please remove `_build` and `.ipynb` files and add to `.gitignore`.

**Comment by yibeichan** [comment]
hello!
@effigies, I removed `_build` folder and `.ipynb` files, added them to `.gitignore`. 
@djarecka I updated the binder link but it didn't work. I think it might automatically link with the repo where the jupyterbook is hosted. So, if we merge this book to `nipype:master`, the link should automatically update.

File: jupyterbook/_config.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Book settings
+# Learn more at https://jupyterbook.org/customize/config.html
+
+title: Pydra Tutorial
+author: TODO
```
Comment (djarecka) [inline jupyterbook/_config.yml:5]:
```suggestion
author: Pydra Developers
```

File: jupyterbook/references.bib [hunk 2]
```diff
@@ -0,0 +1,56 @@
+---
+---
+
+@inproceedings{holdgraf_evidence_2014,
```
Comment (djarecka) [inline jupyterbook/references.bib:4]: do we really use all the references from the file?

Comment (effigies) [reply jupyterbook/references.bib:4]: This is just the default. I doubt we use any, but it's no real problem and it makes it clear where to add citations when we do have them.

File: jupyterbook/_toc.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    - file: notebooks/1_intro_pydra
+    - file: notebooks/2_intro_functiontask
+    - file: notebooks/3_intro_functiontask_state
+    - file: notebooks/4_intro_workflow
+    - file: notebooks/5_intro_shelltask
+    - file: notebooks/6_glm_from_nilearn
+  - caption: About Pydra
+    chapters:
+    - file: about/team
+    - file: about/cite_pydra
```
Comment (effigies) [inline jupyterbook/_toc.yml:18]: Until there's content in these files, may be worth commenting out.

```suggestion
#  - caption: About Pydra
#    chapters:
#    - file: about/team
#    - file: about/cite_pydra
```

**Pull request merged**